src/components/Account/accounNavBar.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../styles/variables.scss";
+
+.NavBarAccount ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  height: 40px;
+  li {
+    color: $VeryLightPink;
+    padding: 8px;
+    border-radius: 8px;
```

Comment:
consejo que te puede ahorrar vida, los border radius, suelen ser variable global ya que se intenta mantener consistencia en ese tipo de atributos

Reply:
I will create this variable right now. This was commit was a test. Beacouse I have to explain how the people could use the first commit has template to start their own projects.